Dashlane/Connected/Session Services/SessionNotificationService.swift

```diff
@@ -74,7 +74,7 @@ extension SessionNotificationService {
         let logger = self.logger
         let settings = self.settings
 
-        webService.sendRequest(to: "/1/devices/setPushNoficationID",
+        webService.sendRequest(to: "/1/devices/setPushNotificationID",
```

Comment:
This presumably would need server side support ...

Reply:
we plan to drop this endpoint anyway on the server side thanks for flagging